Documentation/guides/BuildProcess.md

```diff
@@ -1135,6 +1135,29 @@ server, the following MSBuild properties can be used:
 
     Note: `env:` is not supported when using `AndroidPackageFormat`=`aab`. 
 
+-   **JarsignerTimestampAuthorityUrl** &ndash; This property
+    allows you to specify a Url to a timestamp authority
+    service. This can be used to make sure your apk signature
+    includes a timestamp.
+    See [time-of-signing](https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html) for more details.
+
+    ```xml
+    <PropertyGroup>
+        <JarsignerTimestampAuthorityUrl >http://example.tsa.url</JarsignerTimestampAuthorityUrl>
```

Comment:
Tiny formatting issue: I think the space at the end of `<JarsignerTimestampAuthorityUrl >` is unintentional? I can also easily tidy up that kind of tiny detail when doing a sync to docs.microsoft.com, so it's fine to leave it how it is for now too.